packages/elements/src/card/index.ts

```diff
@@ -184,6 +181,27 @@ export class Card extends BasicElement {
     return this.footerHasContent || !!this.footer;
   }
 
+  /**
+   * Observes attribute change for `attributeChangedCallback`
+   */
+  static get observedAttributes (): string[] {
+    return ['aria-level'].concat(super.observedAttributes);
+  }
+
+  /**
+   * Run when observed attributes change value
+   * @param name attribute name
+   * @param oldValue old attribute value
+   * @param newValue new attribute value
+   * @returns {void}
+   */
+  public attributeChangedCallback (name: string, oldValue: string | null, newValue: string | null): void {
+    super.attributeChangedCallback(name, oldValue, newValue);
+    if (name === 'aria-level') {
+      this.headingLevel = newValue;
```

Comment:
Why do you set aria-level here? The app developer sets the header manually, so they have to define it:
```html
<ef-card>
        <span slot="header"><h5>Header Slot</h5></span>
        <p>Content</p>
      </ef-card>
```

Reply:
The menu should be inside the heading level, not a sibling. what do you think?

Reply:
Not sure, menu belongs to the card, not to the heading.

Reply:
I have already discussed it with the Bangkok team, that it is better if we place the menu outside the header because in the VoiceOver users do not need to press an extra key to go inside the header to access the menu.

Reply:
The next question is do we need to support `aria-level` attribute for `ef-card`? because developers can use slot instead is the same you mention above example.

Reply:
We do not need to support, as per example developers can either provide heading element, or provide aria-level themselves.

Reply:
I think so too, I have already removed the code for aria-level and I will mention this in docs.